# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bdb33/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- db-3.3.11/include/db_cxx.h.vanilla	2008-04-09 10:21:01.000000000 +0000
+++ db-3.3.11/include/db_cxx.h	2008-04-09 10:22:14.000000000 +0000
@@ -49,12 +49,14 @@
 // Forward declarations
 //
 
-#include <iostream.h>
+#include <iostream>
 #include <stdarg.h>
 #include "db.h"
 #include "cxx_common.h"
 #include "cxx_except.h"
 
+using namespace std; // evil, but this is aging code anyway ...
+
 class Db;                                        // forward
 class Dbc;                                       // forward
 class DbEnv;                                     // forward
